adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/commands/CancelTickets.java

```diff
@@ -0,0 +1,16 @@
+package com.microservicesteam.adele.ticketmaster.commands;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface CancelTickets extends TicketsCommand {
```

Comment:
Question is good. From the user point of view yes, - I agree - we would cancel booking instead of individual tickets. However from `Ticket master` point of view I wouldn't introduce responsibility to maintain bookings with tickets. And lookup every tickets belong to a booking in case of a cancel. Probably I would keep it as plain as it could be handling (position - ticket) pairs only. 
Perhaps `BookingClerk` and an `ExpirationController` should be responsible for maintaining the booking and fire `CancelTickets` commands appropriately. 

Hm?